.github/workflows/ci.yml

```diff
@@ -104,7 +104,7 @@ jobs:
         uses: "actions/deploy-pages@v4"
 
   cli-dev:
-    name: "Flox CLI Tests"
+    name: "dev"
```

Comment:
Whether these are much more descriptive for someone without the context (that one group of tests is run in dev mode and the other one is run on remote builders) is debatable but it is an improvement to the geenric names we had before.